Added challenge to chapter 1

dsmits opened the pull request.

hcadavid reviewed (commented):
These are my comments after going through the episode with 'fresh  eyes':

I think it may be useful making the 'Secret Sharing' example more self-explanatory. The table is practical for representing the solution, but may be confusing at first (the reader may get confused and wonder why Mees, Sara and Noor receive their own secrets :) ). Adding a diagram (the one with the f(y1, y2, y3), but with the data in the example (e.g, with the '50' on the jig-saw puzzle piece that Sara receives) will make this more clear and better connected to the theory paragraph. 
It may be also useful, for clarity, to make explicit what is the 'encryption' process in this case (e.g., starting with each one choosing a random number for herself,...)


https://github.com/vantage6/vantage6-workshop/blob/b557766d3139b96db4ad1a496adeb1ba07d6a69d/episodes/chap1_introduction_FL.md?plain=1#L153-L174

The proposed challenge involves a good question! But perhaps it would be better to rephrase it in a way it lead participants to the kind of expected solution (not just a yes/no answer)? Something like: 'If a hacker managed to get access to Sara's computer, where Mees and Noor secrets are stored, would the secrecy of their ages still preserved? What if the hacked get access to Noor's computer also...?

https://github.com/vantage6/vantage6-workshop/blob/b557766d3139b96db4ad1a496adeb1ba07d6a69d/episodes/chap1_introduction_FL.md?plain=1#L178-L181

dsmits commented:
@hcadavid I added an exercise about the hacker that you were suggesting. I kept the old exercise because I'm not trying to make a point about the security of secret sharing. The point is  about whether aggregation in itself preserves privacy. I hope that i have made it more clear in this way.

I also added the pictures you requested.

hcadavid commented:
@dsmits The example and the challenges are indeed quite clear now, they look and read nice! Should I merge the PR? or do you want @frankcorneliusmartin to add his review first?
